pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -141,25 +146,22 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
           </div>
 
           <div className={styles.cardsContainer}>
-            <InfoCard
-              title="Organisatie"
-              content={
-                <>
-                  {_getComponent.data.embedded?.legal.embedded?.repoOwner.name && (
-                    <div>
-                      Dit component wordt aangeboden door{" "}
-                      <span onClick={() => navigate("/organizations/f9d9190e-74f0-4e91-a5d8-0f0e6dad2bd0")}>
-                        <Link icon={<ArrowRightIcon />} iconAlign="start">
-                          {_getComponent.data.embedded?.legal.embedded?.repoOwner.name}
-                        </Link>
-                        .
-                      </span>
-                    </div>
-                  )}
-                  {!_getComponent.data.embedded?.legal.embedded?.repoOwner.name && "Er is geen informatie beschikbaar."}
-                </>
-              }
-            />
+            {TempOrganization.map((organization) => (
+              <OrganizationCard
+                title={{ label: organization.name, href: `/organizations/${organization.id}` }}
+                description={organization.description}
+                website={organization.website}
+                logo={organization.logo}
+                components={{
+                  owned: organization.owns?.length.toString() ?? "0",
+                  supported: organization.supports?.length.toString() ?? "0",
+                  used: organization.uses?.length.toString() ?? "0",
+                }}
+                gitHub={organization.github}
+                gitLab={organization.gitlab}
+                type={organization.type}
+              />
+            ))}
```

Comment:
Right now, we're mapping over an array of organizations where we'll always want to receive one (I think..). If we refactor follow the feedback stated earlier in this review, we could completely remove the `.map()` here, and have the same outcome, like so:

```
<OrganizationCard
  title={{ label: organization.name, href: `/organizations/${organization.id}` }}
  description={organization.description}
  website={organization.website}
  logo={organization.logo}
  components={{
    owned: organization.owns?.length.toString() ?? "0",
    supported: organization.supports?.length.toString() ?? "0",
    used: organization.uses?.length.toString() ?? "0",
  }}
  gitHub={organization.github}
  gitLab={organization.gitlab}
  type={organization.type}
/>
```